package.json

```diff
@@ -28,8 +28,8 @@
   "author": "c0b41 <cobaimelan@protonmail.ch>",
   "devDependencies": {
     "expect.js": "^0.3.1",
-    "mocha": "^2.3.3",
-    "mocha-clean": "^0.4.0"
+    "mocha": "^6.2.2",
+    "mocha-clean": "^1.0.0"
   },
   "homepage": "https://github.com/c0b41/xtorrent"
 }
```

Comment:
can you also add engines property for node version >= 10

Reply:
Node 8 should work fine too!

Reply:
yeah, i know but lts ending no point to support old version https://nodejs.org/en/about/releases/

Reply:
Most people still use it tho and it's not EOL yet! And this is really not that important.

Reply:
any way, i am gonna drop 2020

Reply:
Yeah next year is good to drop version 10